<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panelWindow</class>
 <widget class="QWidget" name="panelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>271</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="changePassword">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(227, 227, 227);
font: 16pt &quot;Oswald&quot;;
color: rgb(0, 0, 0);
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>Change password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="userView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(227, 227, 227);
font: 16pt &quot;Oswald&quot;;
color: rgb(0, 0, 0);
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>UserView</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addNewUser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(227, 227, 227);
font: 16pt &quot;Oswald&quot;;
color: rgb(0, 0, 0);
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>Add new user</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>user login already used, please use another login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
